_posts/2020-11-02-changelog-134.markdown

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #134"
+date: 2020-11-02 02:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6267 -->
+* Support debugging in IDEA Ultimate and RubyMine on Windows.
+  Only MSVC toolchain is supported. Also, [Native Debugging Support](https://plugins.jetbrains.com/plugin/12775-native-debugging-support) plugin is required
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6242 -->
+* Show project loading steps in the Sync tool window. Now it's more convenient to see errors of project loading
+
+    <img src="/assets/posts/changelog-134/sync-tool-window.png" alt="Sync tool window" width="700px"/>
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5646 -->
+* Provide completion for `rustc` and `clippy` [lint attributes](https://doc.rust-lang.org/rustc/lints/levels.html) (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-134/lint-attrs-completion" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6204 -->
+* Support `Go to Declaration` for Cargo features in `Cargo.toml`
+
+    {% include gif-img.html path="/assets/posts/changelog-134/goto-cargo-features" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5871 -->
+* Check if struct fields are not bound multiple times in struct patterns (see [E0025](https://doc.rust-lang.org/error-index.html#E0025)) (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6107 -->
+* Move refactoring (`Refactor | Move` or <kbd>F6</kbd>) improvements:
+    * Support move multiple files/directories
+    * Support move a file to another crate
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6205 -->
+* Clear variables view on `:clear` command in REPL (`Tools | Rust REPL`)
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6279 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6280 -->
```

Comment:
nit: Let's keep these comments just above the corresponding item. It may simplify the search in future